pkg/pdsutils/api/account.go

```diff
@@ -0,0 +1,89 @@
+// Package api comprises of all the components and associated CRUD functionality
+package api
+
+import (
+	"context"
+	status "net/http"
+
+	pds "github.com/portworx/pds-api-go-client/pds/v1alpha1"
+	log "github.com/sirupsen/logrus"
+)
+
+/*
+	Account struct consists the context and apiclient.
+	It can be used to utilize CRUD functionality with respect to Accounts API.
+*/
+type Account struct {
+	Context   context.Context
+	apiClient *pds.APIClient
+}
+
+/*
+	Get the List of all the Accounts.
+	@return []pds.ModelsAccount, error
+*/
+func (account *Account) GetAccountsList() ([]pds.ModelsAccount, error) {
+	client := account.apiClient.AccountsApi
```

Comment:
can you add a first line with logrus.Infof("Getting account list")
And post that use logrus.Debugf instead of info.